dspace-spring-rest/src/main/java/org/dspace/app/rest/converter/DSpaceConverter.java

```diff
@@ -15,7 +16,11 @@ public R convert(M source) {
 		return fromModel(source);
 	}
 
-	public abstract R fromModel(M obj);
+	public R fromModel(M obj) {
+		return fromModel(obj, DSpaceObjectRest.PRJ_DEFAULT);
```

Comment:
I see a potential problem here as I have implemented the Spring Converter interface to easily apply the conversion over a collections of object. See https://github.com/DSpace/DSpace/blob/master/dspace-spring-rest/src/main/java/org/dspace/app/rest/repository/CollectionRestRepository.java#L74

Two options:
- we can switch to inject a ConverterFactory instead of a converter in the repository and set the projection during the converter creation
- we can keep the converter unaware of the projection. This was my initial idea and still preferred way. The converter should convert anything. Not initialized object (to be lazy loaded) should return lazy loadable convered object to be fully load if necessary during the application of the projection. The layer that will apply the projection will so responsible to lazy load required embedded objects and hide unwanted details